src/cache.rs

```diff
@@ -57,7 +57,14 @@ impl<S: CompilerSettings> CompilerCache<S> {
 
     /// Removes entry for the given file
     pub fn remove(&mut self, file: &Path) -> Option<CacheEntry<S>> {
-        self.files.remove(file)
+        if let Some(entry) = self.files.remove(file) {
```

Comment:
`.inspect(|entry| { ... })`

Reply:
https://github.com/foundry-rs/compilers/actions/runs/8923860095/job/24508985313?pr=118#step:5:571

should I bump MSRV?

Reply:
Ah no then use `map`